internal/isoutil/isoutil.go

```diff
@@ -228,3 +232,64 @@ func (h *installerHandler) VolumeIdentifier() (string, error) {
 
 	return strings.TrimSpace(string(volumeId)), nil
 }
+
+func (h *installerHandler) GetFileLocation(filePath string) (int64, error) {
+	fs, err := h.isoFS()
+	if err != nil {
+		return 0, err
+	}
+
+	isoFile, err := fs.OpenFile(filePath, os.O_RDONLY)
+	if err != nil {
+		return 0, errors.Wrapf(err, "Failed to open file %s", filePath)
+	}
+
+	myFile := isoFile.(*iso9660.File)
+	defaultSectorSize := int64(2 * 1024)
+	offset := int64(myFile.Location()) * defaultSectorSize
+	return offset, nil
+}
+
+func (h *installerHandler) Copy(filePath string) error {
+	fsFile, err := os.OpenFile(h.isoPath, os.O_RDONLY, 0o664)
```

Comment:
I was using this convention (fs vs os) in other functions to indicate when something was within the iso vs on the local filesystem (os). Not sure that it really fits this scenario. Maybe `src`/`dest` would be better in this case as both files are on the local filesystem.

Reply:
yeah, src/dst should be reasonable enough i guess.